cypress/features/regression/test/accordion.feature

```diff
@@ -7,7 +7,17 @@ Feature: Accordion component
   @positive
   Scenario Outline: Set Accordion iconType to <iconType>
     When I select iconType to "<iconType>"
-    Then Accordion iconType property on preview is set to "<iconType>"
+    Then Accordion row is "closed" then iconType property on preview is set to "<iconType>"
```

Comment:
```suggestion
    Then Accordion row is "closed" and iconType property on preview is set to "<iconType>"
```

Reply:
Don't agree with you, because CSS are changing depends on the state of accordion row, that's why I used `then` instead of `and`